opendatahub-io/odh-dashboard
# [RHOAIENG-2225] Convert API - List Runs, Create run w/o input params, schedule

Closes: https://issues.redhat.com/browse/RHOAIENG-2225
Starts: https://issues.redhat.com/browse/RHOAIENG-2295

## Description
Added ability to (with v2 endpoints):
    - Create triggered runs, minus input params
    - View triggered runs in a table with all contents in each row
    - Filter by all existing filters in triggered runs table toolbar, where all statuses are updated with newer "state" variables
    - Delete 1 or more triggered runs
    - Stop a triggered run

<img width="1436" alt="image" src="https://github.com/opendatahub-io/odh-dashboard/assets/96431149/21edf00c-7f96-4b47-9fee-1acb032b8420">

## How to test
1. Create 3 triggered runs using the "Create run" button, verify row contents are displayed correctly
2. Stop 1 run
3. Attempt to use all filter options in the dropdown, verify table reflects those filter changes
4. Delete 2 runs
5. Delete the remaining run

## Test Impact
Updated all impacted unit tests, and enabled triggered run delete cypress tests, and updated accordingly. The delete tests verify the table loads with data, then deletes that data, and verifies empty state.

## Request review criteria:
Self checklist (all need to be checked):

After the PR is posted & before it merges:
